### PR TITLE
fix: Add dashboard option `cookieSessionMaxAge` to keep user logged in across browser sessions

### LINKED ISSUE #2365: Login session ends when browser session ends, cookie maxAge ignored
### New Issue Checklist
- [X] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [X] I am not just asking a [question](https://github.com/parse-community/.github/blob/master/SUPPORT.md).
- [X] I have searched through [existing issues](https://github.com/parse-community/parse-dashboard/issues?q=is%3Aissue).
- [X] I can reproduce the issue with the latest versions of [Parse Server](https://github.com/parse-community/parse-server/releases) and [Parse Dashboard](https://github.com/parse-community/parse-dashboard/releases). 

### Issue Description
cookie-session - maxAge property has no effect, at the moment its in an object 

### Steps to reproduce
Just open chrome inspect elements and see that the cookie expires on session

### Actual Outcome
cookie expires on session

### Expected Outcome
cookie will expire in 2 weeks

### Environment

Dashboard
- Parse Dashboard version: `5.0.0`
- Browser (Safari, Chrome, Firefox, Edge, etc.): `Chrome, Firefox (all...)`
- Browser version: ``

The problem is at [Authentication.js row 59](https://github.com/parse-community/parse-dashboard/blob/19f330354ec9757da52178d718596b403ac81ccc/Parse-Dashboard/Authentication.js), maxAge should be a prop of the configuration object and not nested in cookie :{}

See actual API
[https://github.com/expressjs/cookie-session](https://github.com/expressjs/cookie-session)

A feature request for this bug will be to have maxAge property via config

Would love to make a PR...

### PR DESCRIPTION
**Pull request by zivchen**
Added maxAge config for cookie session, cookieMaxAge

Closes: https://github.com/parse-community/parse-dashboard/issues/2365

### CODE REVIEW

File: Parse-Dashboard/Authentication.js [hunk 1]
```diff
@@ -54,14 +54,13 @@ function initialize(app, options) {
   });
 
   var cookieSessionSecret = options.cookieSessionSecret || require('crypto').randomBytes(64).toString('hex');
+  const cookieMaxAge = options.cookieMaxAge;
```
Comment (mtrezza) [inline Parse-Dashboard/Authentication.js]: Maybe we should call it `cookieSessionMaxAge` similar to `cookieSessionSecret`.

**Review by mtrezza — CHANGES_REQUESTED** [review]

File: Parse-Dashboard/Authentication.js (hunk 1 above)
Comment (zivchen) [reply Parse-Dashboard/Authentication.js]: changed all cookieMaxAge cookieSessionMaxAge.
Thx

File: Parse-Dashboard/index.js [hunk 2]
```diff
@@ -28,6 +28,8 @@ program.option('--trustProxy [trustProxy]', 'set this flag when you are behind a
 program.option('--cookieSessionSecret [cookieSessionSecret]', 'set the cookie session secret, defaults to a random string. You should set that value if you want sessions to work across multiple server, or across restarts');
 program.option('--createUser', 'helper tool to allow you to generate secure user passwords and secrets. Use this on trusted devices only.');
 program.option('--createMFA', 'helper tool to allow you to generate multi-factor authentication secrets.');
+program.option('--cookieSessionMaxAge [cookieSessionMaxAge]', 'set the cookie maxAge, defaults to session');
```
Comment (mtrezza) [inline Parse-Dashboard/index.js]:
```suggestion
program.option('--cookieSessionMaxAge [cookieSessionMaxAge]', '(Optional) Sets the time in seconds for when the session cookie will be deleted and the dashboard user has to re-login; defaults to `undefined` which means the cookie will be deleted when the browser session ends.');
```

File: Parse-Dashboard/index.js [hunk 3]
```diff
@@ -28,6 +28,8 @@ program.option('--trustProxy [trustProxy]', 'set this flag when you are behind a
 program.option('--cookieSessionSecret [cookieSessionSecret]', 'set the cookie session secret, defaults to a random string. You should set that value if you want sessions to work across multiple server, or across restarts');
 program.option('--createUser', 'helper tool to allow you to generate secure user passwords and secrets. Use this on trusted devices only.');
 program.option('--createMFA', 'helper tool to allow you to generate multi-factor authentication secrets.');
+program.option('--cookieSessionMaxAge [cookieSessionMaxAge]', '(Optional) Sets the time in seconds for when the session cookie will be deleted and the dashboard user has to re-login; defaults to `undefined` which means the cookie will be deleted when the browser session ends.');
```
Comment (mtrezza) [inline Parse-Dashboard/index.js]:
```suggestion
program.option('--cookieSessionMaxAge [cookieSessionMaxAge]', '(Optional) Sets the time in seconds for when the session cookie will be deleted and the dashboard user has to re-login; if no value is set then the cookie will be deleted when the browser session ends.');
```

File: Parse-Dashboard/server.js [hunk 4]
```diff
@@ -19,6 +19,7 @@ module.exports = (options) => {
   const allowInsecureHTTP = options.allowInsecureHTTP || process.env.PARSE_DASHBOARD_ALLOW_INSECURE_HTTP;
   const cookieSessionSecret = options.cookieSessionSecret || process.env.PARSE_DASHBOARD_COOKIE_SESSION_SECRET;
   const trustProxy = options.trustProxy || process.env.PARSE_DASHBOARD_TRUST_PROXY;
+  const cookieSessionMaxAge = options.cookieSessionMaxAge || process.env.PARSE_DASHBOARD_COOKIE_MAX_AGE;
```
Comment (mtrezza) [inline Parse-Dashboard/server.js]:
```suggestion
  const cookieSessionMaxAge = options.cookieSessionMaxAge || process.env.PARSE_DASHBOARD_COOKIE_SESSION_MAX_AGE;
```

**Review by mtrezza — COMMENTED** [review]
I've made some minor changes, could you review so we can merge this?

**Review by zivchen — COMMENTED** [review]
Looks great, Tnx

**Comment by mtrezza** [comment]
This will appear as a bug fix in the change log, but we may as well have merged it as a new feature; just a note if comes across this and wonders.

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.1.0-alpha.4](https://github.com/parse-community/parse-dashboard/releases/tag/5.1.0-alpha.4)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.1.0-beta.2](https://github.com/parse-community/parse-dashboard/releases/tag/5.1.0-beta.2)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.1.0](https://github.com/parse-community/parse-dashboard/releases/tag/5.1.0)

**Pull request merged**